django/santropolFeast/order/templates/list.html

```diff
@@ -29,12 +29,12 @@
   <tbody>
     {% for obj in order %}
       <tr>
-        <td>{{obj.creation_date}}</td>
-        <td>{{obj.delivery_date}}</td>
-        <td>{{obj.client.member}}</td>
+        <td><a href="{% url 'order:view' id=obj.id %}">{{obj.creation_date}}</a></td>
+        <td><a href="{% url 'order:view' id=obj.id %}">{{obj.delivery_date}}</a></td>
+        <td><a href="{% url 'member:view' id=obj.client.id %}">{{obj.client.member}}</a></td>
```

Comment:
The table shoud include a new column with a View action / Button, rahter than links on multiple column.